test/unit/inet-address-tests.js

```diff
@@ -94,6 +94,19 @@ describe('InetAddress', function () {
       helper.assertInstanceOf(val, InetAddress);
       assert.strictEqual(val.toString(), '10.11.12.13');
     });
+    it('should parse IPv6 Addresses with Embedded IPv4 Addresses', function () {
+      [
+        ['0:0:0:0:0:FFFF:129.144.52.38',    '00000000000000000000ffff81903426'],
+        ['0:0:0:0:0:0:129.144.52.38',       '00000000000000000000000081903426'],
+        ['::ffff:129.144.52.38',            '00000000000000000000ffff81903426'],
+        ['::ffff:254.255.52.32',            '00000000000000000000fffffeff3420'],
+        ['::13.1.68.3',                     '0000000000000000000000000d014403'],
+        ['fe::13.1.68.3',                   '00fe000000000000000000000d014403']
```

Comment:
Hey @jorgebay - To my knowledge this isn't valid. `::ffff:` prefix is valid per RFC while the `::` prefix is deprecated but commonly supported. Non-RFC docu: https://en.m.wikipedia.org/wiki/IPv6#IPv4-mapped_IPv6_addresses